ftp/1.5/modules/ROOT/pages/index.adoc

```diff
@@ -3,68 +3,35 @@
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-FTP Connector v1.5
-
-Anypoint Connector for FTP (FTP Connector) provides access to files and folders on an FTP server. Its main features include:
+Anypoint Connector for FTP (FTP Connector) provides access to files and folders on an FTP server. FTP connector operations manage file transfers over the FTP (File Transfer Protocol) protocol. The connector's main features include:
 
 * The ability to read files or fully list directory contents on demand.
 * Support for common FTP operations such as creating directories and copying, moving, renaming, and deleting files.
 * Support for locking files.
 * File matching functionality.
-* A design that is fully consistent with the File and SFTP connectors. The same set of operations is available on all these connectors, and they behave similarly.
-
-
-== POM File Information
-
-[source,xml,linenums]
-----
-<dependency>
-  <groupId>org.mule.connectors</groupId>
-  <artifactId>mule-ftp-connector</artifactId>
-  <version>RELEASE</version>
-  <classifier>mule-plugin</classifier>
-</dependency>
-----
-
-Mule converts RELEASE to the latest version. To specify a version, view
-https://www.mulesoft.com/exchange/org.mule.connectors/mule-ftp-connector/[FTP Connector] in
-Anypoint Exchange and click *Dependency Snippets*.
+* A design that is consistent with the File and FTP connectors.
```

Comment:
Since "File" is init. capped, it is a connector name, and connector names always capitalize "Connector":
```suggestion
* A design that is consistent with the File and FTP Connectors.
```